test/factory.js

```diff
@@ -32,17 +33,15 @@ describe("factory to create Dimension X", function () {
       "3525A",
       1,
       1000,
-      Signers[1].address,
-      PlatformAddr
+      Signers[1].address
```

Comment:
will have to add delegate/domain server address as well based on the network being tested
https://docs.spanning.network/docs/deployment